readthedocs_theme/templates/base.html

```diff
@@ -48,6 +48,10 @@
     <link href="{{ SITEURL }}/{{ FEED_RSS }}" type="application/atom+xml" rel="alternate" title="{{ SITENAME }} RSS Feed" />
     <script src="{{ SITEURL }}/theme/js/vendor.js"></script>
     <script src="{{ SITEURL }}/theme/js/site.js"></script>
+
+    <!-- Analytics -->
```

Comment:
Just to keep our HTML output nice and clean:

```suggestion
    {# Analytics #}
```

Reply:
I kinda like having the comment in the public output, but this works fine if that's our style here.